graceoncampus/goc-web-v3
# schema updating and implementing rides functionality

- edit schema to remove old Event type (limiting to church for now), modify driver and rider signup forms to reflect changes.
- update schema to implement rides. basic hookup from rides spreadsheet upload to dynamodb update via graphql

to test:
- ask isaac for `goc-form-ca6452f3be85.json`
- run `amplify pull` and `amplify codegen --maxDepth 4` to update graphql models
- test rides upload with sample rides spreadsheet, ensure it works
